src/api/spec/factories/kiwi_image.rb

```diff
@@ -0,0 +1,17 @@
+FactoryGirl.define do
+  factory :kiwi_image, class: Kiwi::Image do
+    name { Faker::Name.name }
+    md5_last_revision { Faker::Crypto.md5 }
+
+    factory :kiwi_image_with_package do
+      transient do
+        package_name { 'package_with_kiwi_image' }
+      end
+
+      after(:create) do |image, evaluator|
+        image.package = create(:package, name: evaluator.package_name)
```

Comment:
It will be also needed a `image.reload`. @Ana06 and I decided to leave it as it is.